src/widgets/radio.jsx

```diff
@@ -441,7 +440,12 @@ var RadioEditor = React.createClass({
                     <PropCheckBox label="Auto-none of the above"
                                   labelAlignment="right"
                                   noneOfTheAbove={this.props.noneOfTheAbove}
-                                  onChange={this.props.onChange} />
+                                  onChange={(e) => {
+                                    var rand = seededRNG(this.props.problemNum)();
+                                    var index = Math.floor(rand * this.props.choices.length);
```

Comment:
This is changing the auto-none-of-the-above semantics slightly. Previously, auto-none-of-the-above would choose the none-of-the-above option at question display/render time, so that a different choice would be the none-of-the-above option each time the question is seen. This chooses the none-of-the-above option at edit time.

We've been having some discussion internally about changing none-of-the-above semantics. The current auto-none-of-the-above semantics are confusing for question writers, so I think it would make sense to get rid of auto-none-of-the-above here and just use the new none-of-the-above this commit adds :).